docs/topics/web-console/proc_discovering-apps-source-platform-migration-view.adoc

```diff
@@ -0,0 +1,47 @@
+:_newdoc-version: 2.15.0
+:_template-generated: 2024-2-21
+:_mod-docs-content-type: PROCEDURE
+
+[id="discovering-apps-source-platform-migration-view_{context}"]
+= Discovering applications through source platform in the Migration view
+
+[role="_abstract"]
+After you configure a source platform, you can discover and import one or more applications to the {ProductShortName} application inventory in the *Migration* view.
+
+.Prerequisites
+
+* You configured at least one source platform instance in the *Administration* view.
+
+* Source platform instance must be running.
+
+.Procedure
+
+. In the {ProductShortName} user interface, select the *Migration* view.
+
+. Click the Options menu ({kebab}) near *Analyze*.
+
+. Click *Discover applications* to open the *Discover applications* window.
++
+The discovery process imports all or selected applications that are deployed in the source platform instance to the *Applications Inventory* page on {ProductShortName}. 
+
+. Click *Add a name* to enter one application per text field and click *+* to add many applications. For example, app-a.
+
+. Click *Add a space* to enter one space per text field and click *+* to add many spaces. For example, space1.
+
+. Click *Add an organization* to enter one organization per text field and click *+* to add many organizations. For example, org1.
++
+
+[NOTE]
+====
+For a CF instance, the application `names` field can contain globs. For example, `app*`.
+====
++
+
+. Click *Next* to review the configuration.
+
+. Click *Discover applications*.
+This action tiggers an application import task.
+
+. Close the *Discover applications* window.
++
+. Optional: Hover over the source platform name to check the status of the application import task and click the application import task to view the task details in YAML format.
```

Comment:
The optional step is not relevant here. The user can check the task status in the Task Manager page (can be added as optional). Once an app is imported it will appear in the application inventory page